cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/sparksql/RowBuilder.java

```diff
@@ -20,9 +20,8 @@
 package org.apache.cassandra.spark.sparksql;
 
 import org.apache.cassandra.spark.data.CqlTable;
-import org.apache.spark.sql.catalyst.expressions.GenericInternalRow;
 
-public interface RowBuilder
+public interface RowBuilder<T>
```

Comment:
Since RowBuilder is now generic, I think the `SchemaFeature` will have to updated according to declare the generic type. Right? The current features are all decorating the spark row.